container/create_image_config.py

```diff
@@ -121,10 +126,34 @@ def Stamp(inp):
     elif '{' in value:
       labels[label] = Stamp(value)
 
+  creation_time = None
+  if args.creation_time:
+    creation_time = Stamp(args.creation_time)
+    try:
+      # If creation_time is parsable as an int type, assume unix epoch timestamp.
```

Comment:
oh good, you handled the breaking behavior soon to arrive in bazel (https://bazel-review.googlesource.com/c/bazel/+/48211).

maybe add a TODO to fix this before 2038. :)

Reply:
T_T
I'll wait until the change lands

Reply:
Should I make this change require bazel 0.12? Or should I just create a reasonable test to support both, e.g. parseable as float -> unix timestamp, >1e10 -> milliseconds, <= 1e10 -> seconds

Reply:
I'd vote for "gracefully" handling both forms of timestamps, rather than requiring bazel 0.12+.

Reply:
Done.